gulpfile.js

```diff
@@ -94,7 +103,7 @@ gulp.task('browserify', function() {
   .transform("babelify", {presets: ["es2015"]})
   .bundle()
   .pipe(source('script.min.js'))
-  .pipe(streamify(uglify()))
+  //.pipe(streamify(uglify()))
```

Comment:
Oops, yes, didn't mean to check in that. Will fix.

Sourcemap would be nice, can perhaps see what I can do later (could make an issue).